### ISSUE TITLE
Individaul Count

### ISSUE THREAD

**Issue by acdoll** [ArctosDB/arctos#3908]
Where does the value for 'individualcount' come from? Is it just a max # of Lot Count in a specimen's parts? Asking due to questions about GBIF flags that read "Occurrence status inferred from Individual Count" (e.g., https://www.gbif.org/occurrence/1041109413), while other specimens in GBIF don't display an Individual Count and have an 'inferred' occurrence status without any flags (e.g., https://www.gbif.org/occurrence/1265236767).

**Comment by Jegelewicz** [comment]
I can't say for sure, but I think that the fact that you have an egg (and not a bird) causes this to happen.

**Comment by dustymc** [comment]
See https://github.com/ArctosDB/arctos/issues/3901

I'm not setting that at all except for ento and fish, but I'm not un-setting it either so that value is probably from some very old (and busted, because we stopped for some reason!) calculation.

**Comment by Jegelewicz** [comment]
But I do not know where "individual count" is coming from, because it appears that it should be 5 for the egg record....

**Comment by acdoll** [comment]
_Should_ be 5, but that's our problem - it was only entered as remarks and not a part count. I think we had decided these needed ground truthing before entering them properly due to previous examples of bad notes. 

I had seen 3901, which is probably where my curiosity came from. Our DMNS:Inv also uses lots; I'm not sure about other marine invert collections, but it might be nice to have that work like the ento and fish (not really asking for any action here, but if you're messing with how this works, it is something to consider). 

FWIW, when you look at 'individualcount' on the specimen results page (the only place I can find it) all of our collections have a value of 1 for most records, but each collection has a small percentage with no value in that field. Any thoughts on why that may be?

**Comment by dustymc** [comment]
>Any thoughts on why 

"I'm not setting that at all except for ento and fish, but I'm not un-setting it either" Things with some value outside those collections are (probably) older, things without are newer (or have been more processed for some reason, or something). 

I suppose we should do something other than ignore it.....

What we're doing now is expensive and probably BARELY meaningful at best - I'm not convinced that doing a bunch of obscure operations based on preferred code tables is entirely useful.

My preference would be to make this y'all's problem via a new attribute or field or something, since it seems to be heavily dependent on various local/procedural/something factors.

My second choice would be some universal calculation that doesn't require bad assumptions and new code for new collection types and such.

If we can't get to any of that, I can just add an "else NULL" to the flat updater.

**Comment by Jegelewicz** [comment]
https://dwc.tdwg.org/terms/#dwc:individualCount

> The number of individuals present at the time of the Occurrence.

> My preference would be to make this y'all's problem via a new attribute or field or something, since it seems to be heavily dependent on various local/procedural/something factors.

I agree because using part counts will create results that are all over the place. Organism ID could be the calculus for this. But also, we currently send the GUID as an Organism ID, which doesn't make sense given the definition.

![image](https://user-images.githubusercontent.com/5725767/132565409-d347486a-82ae-402c-9d55-48e4307d7448.png)

https://dwc.tdwg.org/terms/#dwc:organismID

> An identifier for the Organism instance (as opposed to a particular digital record of the Organism). May be a globally unique identifier or an identifier specific to the data set.

**Comment by acdoll** [comment]
> make this y'all's problem via a new attribute or field or something

Agreed. Trying to algorithm it out based on all our different ways of capturing data seems destined to fail.

**Comment by sharpphyl** [comment]
I know I'm late to the party, but is there something we can do in DMNS:Inv to record specimen count so it translates accurately to GBIF?

Our record DMNS:Inv:32550 [https://arctos.database.museum/guid/DMNS:Inv:32550] contains 230 (very tiny) shells.

<img width="704" alt="Screen Shot 2021-09-18 at 10 28 05 AM" src="https://user-images.githubusercontent.com/15368365/133895783-1ae83702-ed78-4c3c-a44e-679cb23fda62.png">

But in [GBIF](https://www.gbif.org/occurrence/2565925490)  that translates to an Individual Count of 1.  For microshells, that's a loss of what might be helpful information.

<img width="1003" alt="Screen Shot 2021-09-18 at 10 31 54 AM" src="https://user-images.githubusercontent.com/15368365/133895805-ca417c19-5672-41b8-b2cb-57e921197478.png">

**Comment by dustymc** [comment]
https://github.com/ArctosDB/arctos/issues/3908#issuecomment-915469617

**Comment by sharpphyl** [comment]
> I'm not setting that at all except for ento and fish..

Are you saying that you could send the value in Qty to GBIF for DMNS:Inv and have it appear as the Individual count?
